### PR TITLE
Remove dots from inactive radio according to docs

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #702 
[Problem] Wearable radio has new design
[Solution] Update it

Signed-off-by: Kornelia Kobiela <k.kobiela@samsung.com>

### CODE REVIEW

**Comment by singa2000** [comment]
@korneliakobiela 
Would you update the PR, please?

**Comment by TomaszLukawskiSam** [comment]
@korneliakobiela 
Is this PR ready to review?

**Comment by korneliakobiela** [comment]
@TomaszLukawskiSam now is ready to review.

File: src/css/profile/wearable/changeable/theme-circle/checkboxradio.less [hunk 1]
```diff
@@ -139,7 +140,7 @@ input[type="radio"] {
 		}
 	}
 	&:disabled {
-		background: radial-gradient(transparent, transparent 59%, @color_checkboxradio_radiobox_line_off_dim 63%, @color_checkboxradio_radiobox_line_off_dim 68%, transparent 72%, transparent);
+		background: radial-gradient(transparent, transparent 63%, @color_checkboxradio_radiobox_line_off_dim 65%, @color_checkboxradio_radiobox_line_off_dim 68%, transparent 72%, transparent);
```
Comment (singa2000) [inline src/css/profile/wearable/changeable/theme-circle/checkboxradio.less:143]: Is it right  @color_checkboxradio_radiobox_line_off_dim ?
It was defined by @color_checkboxradio_radiobox_line_off: fade(@controller-off-color, 80%); in theme,changeable.less.
It looks different value in GUI Guide.

Comment (TomaszLukawskiSam) [reply src/css/profile/wearable/changeable/theme-circle/checkboxradio.less:143]: I verified color also. In my opinion color of disabled radio off should be #8f8f8f / 24% - current value is #8f8f8f / 30%

Comment (TomaszLukawskiSam) [reply src/css/profile/wearable/changeable/theme-circle/checkboxradio.less:143]: I verified again. There is correct color #8f8f8f / 24%.

Comment (TomaszLukawskiSam) [reply src/css/profile/wearable/changeable/theme-circle/checkboxradio.less:143]: :) . I have to correct my comment again. Valid color for disabled radio off is: #8f8f8f / 30% according to html guideline.

Comment (singa2000) [reply src/css/profile/wearable/changeable/theme-circle/checkboxradio.less:143]: Thanks for your checking ^^.

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

File: src/css/profile/wearable/changeable/common/checkboxradio.less [hunk 2]
```diff
@@ -16,17 +16,6 @@ input[type="radio"] {
 	.mask-size(100% 100%);
 }
 
-input[type="radio"]::after {
```
Comment (TomaszLukawskiSam) [inline src/css/profile/wearable/changeable/common/checkboxradio.less]: I not sure this change.

**Comment by lslachciak** [comment]
@korneliakobiela ready to merge?

**Pull request merged**